src/kmsClientIdAuthorizer.js

```diff
@@ -11,17 +10,15 @@ class kmsClientIdAuthorizer {
     this.clientId = clientId;
     this.encryptedClientSecret = encryptedClientSecret;
     this.defaultRegion = defaultRegion;
+    this.KMS = new AWS.KMS(this.defaultRegion ? { region: this.defaultRegion} : {});
```

Comment:
Shouldn't we default to `eu-west-1`?

Reply:
So, this is not needed because:
 - when run on AWS it will default to the current region
 - this is mostly intended for local use during development